src/main/resources/application-k8s.yml

```diff
@@ -9,13 +9,26 @@ bb-fuel:
     auth: http://edge.${environment.name}.${environment.domain}/api/auth/login
     tokenconverter: http://tokenconverter.${environment.name}.${environment.domain}
   dbs:
-    approvals: http://approvals.${environment.name}.${environment.domain}
+    approvals: http://approvals.${environment.name}.${environment.domain}/integration-api
     transactions: http://transactions.${environment.name}.${environment.domain}
-    contactmanager: http://contacts.${environment.name}.${environment.domain}
-    messages: http://messages.${environment.name}.${environment.domain}
+    contactmanager: http://contacts.${environment.name}.${environment.domain}/integration-api
+    messages: http://messagecenter.${environment.name}.${environment.domain}/integration-api
```

Comment:
can we have something like this for all the services, instead of having to add the 'client-api' or 'integration-api' on the rest clients? So we can have a single source of truth instead of going through every single client?

Reply:
Can we add this simplification later?